Firestore/Example/Tests/Integration/FSTStreamTests.m

```diff
@@ -38,17 +38,25 @@ - (void)writesFinishedWithError:(NSError *_Nullable)error;
  */
 @interface FSTStreamStatusDelegate : NSObject <FSTWatchStreamDelegate, FSTWriteStreamDelegate>
 
+- (instancetype)initFrom:(XCTestCase *)testCase
```

Comment:
Naming: first part of the label should end with "test case".

Also avoid connecting words in the labels for subsequent arguments unless they're critical to the meaning. It's appropriate in e.g. `copyToFile:fromFile` but not here, where you're just specifying the queue argument.

 This should be: `initWithTestCase:(XCTestCase *)testCase queue:(FSTDispatchQueue *)queue`.

Reply:
This was inspired by the name in FEventTester. I updated it to use the naming you suggested and removed the 'using' part.